### ISSUE TITLE
Doesn't open on Windows 10

### ISSUE THREAD

**Issue by NunoMouraz** [mifi/lossless-cut#61]
v1.9.0

Tried x32 and x64, it just basically closes itself automatically or something, it shows up on task manager for a split second and then vanishes.

Tried compatibility mode, executing as admin and to no avail.

Please fix.
--Thanks

**Comment by malte1989** [comment]
Check again, I have no problem with Windows x64 system. 
I had just problems with the start script but solved them with the following code:

`{
  "name": "lossless-cut",
  "version": "1.9.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "start": "electron dist",
    "watch": "npm run build && babel src -d dist --copy-files -w",
    "build": "rd /s /q dist && babel src -d dist --copy-files && mklink /d ..\\node_modules dist\\ && mklink /d ..\\package.json .\\dist\\",
    "download-ffmpeg": "bash ./scripts/ffmpeg-dl/dl.sh",
    "extract-ffmpeg": "bash ./scripts/ffmpeg-dl/extract.sh",
    "copy-ffmpeg": "rd /s /q dist/ffmpeg && mkdir dist/ffmpeg && cp ffmpeg-tmp/binaries/${PLATFORM}_${ARCH}/* dist/ffmpeg",
    "package-single": "npm run copy-ffmpeg && electron-packager dist LosslessCut --out=package --asar.unpackDir=ffmpeg --overwrite --platform=${PLATFORM} --arch=${ARCH} --icon=icon-dist/${ICON} --app-copyright='Copyright (c) 2017 Mikael Finstad' --win32metadata.CompanyName=mifi --win32metadata.FileDescription=LosslessCut --win32metadata.OriginalFilename=LosslessCut.exe --win32metadata.ProductName=LosslessCut --win32metadata.InternalName=LosslessCut",
    "package:darwin_x64": "PLATFORM=darwin ARCH=x64 ICON=app.icns npm run package-single",
    "package:win32_ia32": "PLATFORM=win32 ARCH=ia32 ICON=app.ico npm run package-single",
    "package:win32_x64": "PLATFORM=win32 ARCH=x64 ICON=app.ico npm run package-single",
    "package:linux_ia32": "PLATFORM=linux ARCH=ia32 ICON=app.ico npm run package-single",
    "package:linux_x64": "PLATFORM=linux ARCH=x64 ICON=app.ico npm run package-single",
    "zip": "(cd package && rm -f LosslessCut-*.zip && for f in LosslessCut-*; do zip -r --symlinks \"$f\".zip \"$f\"; done)",
    "icon-gen": "icon-gen -i src/icon.svg -o ./icon-dist -r",
    "package": "npm run package:darwin_x64 && npm run package:win32_ia32 && npm run package:win32_x64 && npm run package:linux_ia32 && npm run package:linux_x64 && npm run zip",
    "release": "gh-release -a package/LosslessCut-darwin-x64.zip,package/LosslessCut-win32-ia32.zip,package/LosslessCut-win32-x64.zip,package/LosslessCut-linux-ia32.zip,package/LosslessCut-linux-x64.zip",
    "gifify": "gifify -p 405:299 -r 5@3 Untitled.mov-00.00.00.971-00.00.19.780.mp4",
    "lint": "eslint --ext .jsx --ext .js .",
    "clean": "rd /s /q dist ffmpeg-tmp/extracted ffmpeg-tmp/binaries package",
    "clean-ffmpeg": "rd /s /q ffmpeg-tmp/archives"
  },

...

`

**Comment by TechManiacHD** [comment]
NunoMouraz - Do you use MSI Afterburner with RivaTuner by any change? I notices that those two make Losseless Cut crashing. Also PowerDirector crashes with them running in the background (program for editing that I use).

**Comment by NunoMouraz** [comment]
No I don't use RivaTuner. I still can't start it. Maybe some sort of coding library is required? (Visual C++, Basic, Python...)

**Comment by TechManiacHD** [comment]
Well... I have been using it on a few different machines and never had any problem. Do you have all updates from windows installed?

**Comment by malte1989** [comment]
But how did you get it managed without replacing the linux based commands like "rm -rf" or "ln -s"?

**Comment by CaptaPraelium** [comment]
Works fine, with rivatuner and afterburner, too... No modifications to the start script. Just works out of the box. Sorry :(

**Comment by Avasam** [comment]
I'm having the exact same issue. It just suddenly stopped working. May appear for a split second in Task Manager. No crash log, nothing in the event viewer.
Things I've done in the last week: Update my graphics driver, update my BIOS, move my OS from an SSD to an M.2 Drive, update a bunch of motherboard-related drivers, reinstall premiere pro.

**Comment by mifi** [comment]
Could u try to run it from the command line and see if any error? Or try to download electron version 8 and run it and see if that crashes too
https://github.com/electron/electron/releases/tag/v8.4.0

**Comment by Avasam** [comment]
I had tried from command line as well. Today it started working again.
Oh, I know what happened, I fixed the permissions of my TEMP folder yesterday (somehow I was no longer in the allowed users), because I noticed another app, which I know creates an executable in TEMP, was having issues.

So I guess LosslessCut just failed silently in that case.
![image](https://user-images.githubusercontent.com/1350584/112667140-46f6fc80-8e33-11eb-920a-53b9166ebc32.png)
